doc/userguide/upgrade.rst

```diff
@@ -128,6 +128,8 @@ Major changes
     can be used to tune this value for TPACKET_V2. Due to the
     increased block size, memory usage has been increased, but
     should not be an issue in most cases.
+- New rule keyword ``entropy`` for alerting based on entropy values. See
```

Comment:
please remove, this doc is not meant to list new features

Reply:
caused a conflict in staging, so needs to be resolved after the next merge. Resolved by removing the upgrade.rst addition.

Reply:
Will remove.